source/roadmap.html.erb

```diff
@@ -17,66 +17,78 @@ title: Roadmap
         <div class="govuk-inset-text">
           <p class="govuk-body">
             This is a record of what we've done, and a plan for what we want to improve next. Our plans can change based on what we learn from testing the product and speaking to
-            our users. We work in 3-monthly cycles.
+            our users.
           </p>
         </div>
         <h2 class="govuk-heading-l">Recently released</h2>
-        <h3 class="govuk-heading-m">Recurring card payments</h3>
+        <h3 class="govuk-heading-m">Accessibility</h3>
         <p class="govuk-body">
-          The first service has started using
-          <%= link_to 'recurring payments', 'https://docs.payments.service.gov.uk/recurring_payments/', class: 'govuk-link' %> through GOV.UK Pay. Kent County Council are using it
-          for their Trading Standards Checked service. This makes it simpler for people to pay for things regularly and easier for service teams to collect payments. If you're
-          interested in using recurring payments, get in touch.
+          We made changes to Pay to meet the Web Content Accessibility Guidelines 2.2 (WCAG 2.2), so that the product can be used by as many people as possible. We have:
         </p>
-        <h3 class="govuk-heading-m">Digital wallets</h3>
+        <ul class="govuk-list govuk-list--bullet">  
+        <li>updated the   
+    <%= link_to 'prefilled payment fields', 'https://docs.payments.service.gov.uk/optional_features/prefill_user_details/#prefill-payment-fields', class: 'govuk-link' %> and 
+    <%= link_to 'take a payment', 'https://docs.payments.service.gov.uk/making_payments/#creating-a-payment', class: 'govuk-link' %> pages in our tech docs, so that public sector services know how to meet the WCAG 2.2 redundant entry requirement</li>
+          <li>upgraded Pay to version 5.0 of the GOV.UK Design System, so that we’re using the most up to date design patterns</li>
```

Comment:
```suggestion
          <li>upgraded GOV.UK Pay to version 5.0 of the GOV.UK Design System, so that we’re using the most up-to-date design patterns</li>
```